src/HotChocolate/Core/src/Validation/Rules/MaxExecutionDepthVisitor.cs

```diff
@@ -42,6 +42,12 @@ public MaxExecutionDepthVisitor(IMaxExecutionDepthOptionsAccessor options)
         FieldNode node,
         IDocumentValidatorContext context)
     {
+        if (_options.SkipIntrospectionFields && 
```

Comment:
This also skips the __typename field. Is this intended?

Reply:
yes its intended

Reply:
All introspection